src/Extensions/Microsoft.Diagnostics.Monitoring.Extension.Common/Utilities.cs

```diff
@@ -0,0 +1,20 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Microsoft.Extensions.Logging;
+
+namespace Microsoft.Diagnostics.Monitoring.Extension.Common
+{
+    internal static class Utilities
+    {
+        public static ILogger CreateLogger<T>() where T : class, IEgressProviderOptions, new()
```

Comment:
This creates a logger from a source that is immediately disposed. For this particular logger, it won't be a crashing issue (it might be for others). It however basically just writes the message to the console directly.

However, this doesn't seem to be a new issue introduced by this change, so this is okay for now.